content/docs/setup/customize/aws-elb/nlb.md

```diff
@@ -25,15 +27,6 @@ Keep in mind the following considerations when working with an NLB:
    
 ## Step 2: Deploy your gateway proxy
 
-Depending on the annotations that you use on your gateway proxy, you can configure the NLB in different ways. 
```

Comment:
All of the steps except for 1 were the same between the two types, so consolidated down the tabs.